subprojects/docs/src/docs/userguide/running-builds/introduction/build_file_basics.adoc

```diff
@@ -0,0 +1,189 @@
+
+====
+[.multi-language-sample]
+=====
+.build.gradle.kts
+[source,kotlin]
+----
+plugins {
+    id("org.jetbrains.kotlin.jvm") version "1.9.0"                      // <1>
+}
+
+repositories {
+    mavenCentral()                                                      // <2>
+}
+
+dependencies {
+    testImplementation("org.jetbrains.kotlin:kotlin-test-junit5")       // <3>
+    testImplementation("org.junit.jupiter:junit-jupiter-engine:5.9.3")
+    testRuntimeOnly("org.junit.platform:junit-platform-launcher")
+    implementation("com.google.guava:guava:32.1.1-jre")
+}
+
+application {
+    mainClass.set("main")                                               // <4>
+}
+
+tasks.named<Test>("test") {
+    useJUnitPlatform()                                                  // <5>
+}
+----
+<1> Add plugins.
+<2> Add dependency location.
+<3> Add dependencies.
+<4> Use convention properties.
+<5> Register or configure tasks.
+=====
+
+[.multi-language-sample]
+=====
+.build.gradle
+[source,groovy]
+----
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.9.0'                       // <1>
+}
+
+repositories {
+    mavenCentral()                                                      // <2>
+}
+
+dependencies {
+    testImplementation 'org.jetbrains.kotlin:kotlin-test-junit5'        // <3>
+    testImplementation 'org.junit.jupiter:junit-jupiter-engine:5.9.3'
+    testRuntimeOnly 'org.junit.platform:junit-platform-launcher'
+    implementation 'com.google.guava:guava:32.1.1-jre'
+}
+
+application {
+    mainClass = 'main'                                                  // <4>
+}
+
+tasks.named('test') {
+    useJUnitPlatform()                                                  // <5>
+}
+----
+<1> Add plugins.
+<2> Add dependency location.
+<3> Add dependencies.
+<4> Use convention properties.
+<5> Register or configure tasks.
+=====
+====
+
+=== 1. Add plugins
+Plugins are bundled tasks used to extend Gradle.
+
+Adding a plugin to a build is called _applying_ a plugin.
+
+[source]
+----
+plugins {
+    id("org.jetbrains.kotlin.jvm") version "1.9.0"
+}
+----
+
+The Kotlin gradle plugin, version 1.9.0, has been applied to the project so that source code written in Kotlin can be used.
+
+=== 2. Add dependency location
+A project generally has several dependencies it needs to do its work.
+Many of the dependencies will be used by the source code.
+
+In the example, the link:https://mvnrepository.com/[Maven Central^] repository has been added as the location for all the binaries that Gradle will need to download to build the project.
+
+[source]
+----
+repositories {
+    mavenCentral()
+}
+----
+
+In this example, the Kotlin JVM Plugin, `org.jetbrains.kotlin.jvm`, will be downloaded by Gradle from the link:https://mvnrepository.com/artifact/org.jetbrains.kotlin.jvm/org.jetbrains.kotlin.jvm.gradle.plugin[Maven Central Repository].
+
+=== 3. Add dependencies
+Dependencies are grouped in configurations and can be retrieved and uploaded from repositories.
```

Comment:
```suggestion
Dependencies are grouped in dependency configurations and are retrieved from repositories defined in the `repositories` block.
```
`retrieved and uploaded` sound wrong to me. To retrieve is to pull dependencies from a repo, to upload is to push built files to a repo after a build. 

The repositories defined in the top-level `repositories` block is only for retrieving/downloading.